Autopilot/PathManager/Src/landingManager.cpp

```diff
@@ -0,0 +1,134 @@
+#include "waypointManager.hpp"
+#include "landingManager.hpp"
+#include <math.h>
+#include "vectorClass.hpp"
+
+const static int SET_THROTTLE_OFF = -1000;
+const static double DISTANCE_OF_LANDING = 10; //in meters
+const static int METERS_PER_DEG_LAT = 111320;
+const static int ANGLE_OF_LANDING = 5; //in degrees
+const static int STALL_SPEED_NO_PACKAGE = 20; //in km/h
+const static int STALL_SPEED_WITH_PACKAGE = 30; //in km/h
+const static double PI = 3.14159265358979323846;
+const static int FLARE_ALTITUDE = 2; //in meters
+const static double DECRAB_ALTITUDE = 0.7; //in meters
```

Comment:
Curious are some of these constants redefined twice?

Reply:
oh yeah true, fixed!